docker-compose.ci.yml

```diff
@@ -0,0 +1,7 @@
+version: '2.1'
```

Comment:
I think this can probably just be in `docker-compose.test.yml` -- that's only used in CI